libass/ass.c

```diff
@@ -1528,6 +1528,7 @@ int ass_track_set_feature(ASS_Track *track, ASS_Feature feature, int enable)
 #else
         [ASS_FEATURE_BIDI_BRACKETS] = NULL,
 #endif
+        [ASS_FEATURE_ALWAYS_WRAP] = &track->always_wrap,
```

Comment:
I think we’re currently preferring to put new fields in `track->parser_priv`, like the `bidi_brackets` above, to minimize ABI breakage risk now and in the future (if we ever decide to change the Feature), since `ASS_Track` is unfortunately public, even if we’ve added a comment that new fields may be appended.

Reply:
Sure thing. I wasn't quite sure, since it seemed like parser_priv seemed off (since this is rendering rather than parser), but it's certainly possible to do so. :-)

Reply:
I do agree about the parsing vs rendering name! I think we might want to rename it at some point, or split, or something. But for now that’s where we’re putting these things.